app/src/main/java/com/release/keyneez/data/model/response/ResponseGetSearchDto.kt

```diff
@@ -0,0 +1,18 @@
+package com.release.keyneez.data.model.response
+
+import kotlinx.serialization.SerialName
+
+@kotlinx.serialization.Serializable
+data class ResponseGetSearchDto(
+    @SerialName("content_key")
+    val key: Int,
+    @SerialName("content_title")
+    val title: String,
+    @SerialName("start_at")
+    val start: String?,
+    @SerialName("end_at")
+    val end: String?,
+    @SerialName("content_img")
+    val background: String?,
+    val liked: Boolean
```

Comment:
```suggestion
    @SerialName("liked")
    val liked: Boolean
```
앱 난독화에서 예외 상황들을 예방하기 위해서 모든 속성에 SerialName을 넣어주는 것이 좋다고 합니다!